pillar/base/haproxy.sls

```diff
@@ -67,6 +67,22 @@ haproxy:
       verify_host: planet.psf.io
       check: "HEAD / HTTP/1.1\\r\\nHost:\\ planet.psf.io"
 
+    {% for tracker, config in pillar["bugs"]["trackers"].items() %}
+    {{ tracker }}:
+      domains:
+        - {{ config.server_name }}
+      verify_host: bugs.psf.io
+      check: "HEAD / HTTP/1.1\\r\\nHost:\\ {{ config.server_name }}"
+    {% endfor %}
+
+    pypy-web:
```

Comment:
looks like this returned-from dead in the rebase.

Reply:
[578429e](https://github.com/python/psf-salt/pull/393/commits/578429eae608e309c550b7397e418f0e9dbae636)